### DISCUSSION TITLE
Constant "This workspace contains a TypeScript version. Would you like to use the wor..." message after latest plugin updates

### DISCUSSION THREAD

**Discussion by SargisPlusPlus** [LazyVim/LazyVim#3365] [Q&A] [answered]
I am working in a team project which contains `.vscode` folder at the root with `settings.json` file with the following content:
```
  "typescript.tsdk": "node_modules/typescript/lib",
  "typescript.enablePromptUseWorkspaceTsdk": true,
```
I recently updated all my plugins and now every time I open the project, I am greeted with the following message:
![Screenshot 2024-05-29 at 10 00 31 PM](https://github.com/LazyVim/LazyVim/assets/6013752/93548a8d-c012-482b-975f-0c74172eb78b)

This started happening recently after I updated my plugins. I think it is `vtsls` plugin that started causing this.
Regardless of the option I choose from above, it keeps coming back whenever I reopen nvim. How do I disable this?

**Comment by roryhen** [comment, marked as answer]
Adding this to my config worked for me:
```lua
  {
    "neovim/nvim-lspconfig",
    ---@class PluginLspOpts
    opts = {
      servers = {
        vtsls = {
          settings = {
            vtsls = { autoUseWorkspaceTsdk = true },
          },
        },
      },
    },
  }
```

**Reply by SargisPlusPlus** [reply]
Thank you
